### PR TITLE
feature(tj): add MoE expert selection stats and gradient conflict monitoring

### PR DESCRIPTION
**Pull request by tAnGjIa520**
- Add MoE expert selection statistics with multi-window sliding buffers and TensorBoard heatmaps
- Add gradient conflict monitoring for encoder, MoE, shared expert, and individual experts
---

- 增加 MoE 专家选择统计，支持多窗口滑动缓冲和 TensorBoard 热力图
- 增加 encoder、MoE、shared expert 及各个 expert 的梯度冲突监控

**实验结果**

moe_expert_selection_wasserstein_distance
image

moe_expert_selection_js_divergence
image

gradient_conflict_comparison_moe_vs_nomoe
image

expert_selection_heatmaps
image

### CODE REVIEW

File: lzero/policy/utils.py [hunk 1]
```diff
@@ -800,3 +800,529 @@ def mz_network_output_unpack(network_output: Dict) -> Tuple:
...
+    
+    assert all(g.shape == gradients[0].shape for g in gradients), "All gradients must have same shape"
+    
+    device = gradients[0].device
+    
+    # Vectorized: stack and normalize all gradients
+    stacked_grads = torch.stack([g.flatten() for g in gradients])
+    normalized_grads = F.normalize(stacked_grads, p=2, dim=1)
+    
+    # 一次性计算余弦相似度矩阵
```
Comment (puyuan1996) [inline lzero/policy/utils.py]: Let's stick to one language. For example, use English for all comments.

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py [hunk 2]
```diff
@@ -135,6 +135,17 @@ def create_config(
                     encoder_type='vit',
                     device='cuda',
                     game_segment_length=20,
```
Comment (puyuan1996) [inline zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py:451]: Let's add two READMEs (Chinese and English) to this branch explaining how to use it to reproduce the relevant charts from the ScaleZero paper.

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py [hunk 3]
```diff
@@ -238,13 +249,13 @@ def create_env_manager() -> EasyDict:
 
     # ==================== Main Experiment Settings ====================
     num_games = 8  # Options: 3, 8, 26
-    num_layers = 2
+    num_layers = 1  # Transformer depth (reduced for faster iteration)
```
Comment (puyuan1996) [inline zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py:566]: Use an option to switch between the debug config and the ScaleZero reproduction config.

**Review by puyuan1996 — CHANGES_REQUESTED** [review]

File: zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py (hunk 2 above)
Comment (puyuan1996) [reply zoo/atari/config/atari_unizero_multitask_segment_ddp_config.py:451]: Please also include the main result images in both the README and the PR description.
